manifests/n/Ngrok/Ngrok/3.1.1/Ngrok.Ngrok.locale.en-US.yaml

```diff
@@ -0,0 +1,32 @@
+# Created with YamlCreate.ps1 v2.2.2 $debug=NVS0.CRLF.5-1-19041-2364.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Ngrok.Ngrok
+PackageVersion: 3.1.1
+PackageLocale: en-US
+Publisher: ngrok, Inc.
+PublisherUrl: https://ngrok.com
+# PublisherSupportUrl:
+PrivacyUrl: https://ngrok.com/privacy
+Author: ngrok, Inc.
+PackageName: Ngrok
+PackageUrl: https://ngrok.com
+License: Proprietary
+# LicenseUrl:
```

Comment:
```suggestion
LicenseUrl: https://ngrok.com/tos
```

Reply:
Note for the future: ngrok has a [copyright link](https://ngrok.com/copyright) as pointed [here](https://ngrok.com/tos#:~:text=https%3A//ngrok.com/copyright) that is currently down. If it ever comes back online, we can use it in the manifest.

![image](https://user-images.githubusercontent.com/12696894/218344855-9b99dd03-a016-4c01-8572-451f41837188.png)